cmd/director_serve.go

```diff
@@ -58,8 +58,8 @@ func serveDirector( /*cmd*/ *cobra.Command /*args*/, []string) error {
 		if err := director.AdvertiseOSDF(); err != nil {
 			panic(err)
 		}
+		go director.PeriodicCacheReload()
```

Comment:
lol Looking at this code yesterday I was wondering why we wanted to talk to topology if we are not running in `osdf` mode. This explains.